recomp-verify.py

```diff
@@ -45,26 +55,29 @@ def verify(spec, cfg, cust, naive, verbose):
         cmd_args.append("--naive")
     if verbose:
         cmd_args.append("--verbose")
+    
+    print("Command to run:", ' '.join(cmd_args))  # Confirm the command with --cust
```

Comment:
is this a debugging print? if so, please delete or make a part of verbose printing